src/Internal/ChannelAtomicGet.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Thesis\Amqp\Internal;
+
+use Amp\Cancellation;
+use Amp\Future;
+use Thesis\Amqp\DeliveryMessage;
+use Thesis\Amqp\Internal\Delivery\Receiver;
+use Thesis\Amqp\Internal\Io\AmqpConnection;
+use function Amp\async;
+
+/**
+ * @internal
+ */
+final class ChannelAtomicGet
```

Comment:
Most of the things in Internal relate to the channel, however they are not all prefixed with `Channel`. Let's call it `AtomicGet`?